onedal/linear_model/incremental_linear_model.py

```diff
@@ -93,7 +94,7 @@ def partial_fit(self, X, y, queue=None):
         hparams = get_hyperparameters("linear_regression", "train")
         if hparams is not None and not hparams.is_default:
             self._partial_result = module.partial_train(
```

Comment:
is there a reason module is used here but deleted from other algos?

Reply:
it is used here twice, that's why i kept it here for the code's brevity